tests/test_openclhpp.cpp

```diff
@@ -2149,6 +2149,220 @@ void testBuildProgramSingleDevice(void)
     TEST_ASSERT_EQUAL(errcode, CL_SUCCESS);
 }
 
+static cl_int clGetProgramInfo_forBuildLog(
+    cl_program         program,
+    cl_program_info    param_name,
+    size_t             param_value_size,
+    void *             param_value,
+    size_t *           param_value_size_ret,
+    int num_calls)
+{
+    (void) num_calls;
+
+    TEST_ASSERT_EQUAL(param_name, CL_PROGRAM_DEVICES);
```

Comment:
Fixes a CI error due to an unused parameter:

```suggestion
    TEST_ASSERT_EQUAL(program, make_program(0));
    TEST_ASSERT_EQUAL(param_name, CL_PROGRAM_DEVICES);
```